app/ui/warehouse_utils.py

```diff
@@ -42,12 +43,12 @@ def convert_time_str(time_str) -> datetime.time:
 
 
 def set_enabled(wh_name: str, enabled: bool):
-    with connection.Connection.get() as conn:
-        _ = conn.sql(
+    with connection.Connection.get() as conn, transaction(conn) as txn:
```

Comment:
might be nice to put this in a helper context manager

Reply:
Good idea.